IPython/__init__.py

```diff
@@ -29,10 +29,10 @@
 #-----------------------------------------------------------------------------
 
 # Don't forget to also update setup.py when this changes!
-v = sys.version_info
-if v[:2] < (3,3):
-    raise ImportError('IPython requires Python version 3.3 or above.')
-del v
+if sys.version_info < (3,3):
+    raise ImportError('IPython 6.0+ requires Python version 3.3 or above. '
+                      'For a version compatible with Python 2.7, please install'
+                      ' the 5.x Long Term Support version.')
```

Comment:
```
raise ImportError('IPython 6.0+ does not support Python 2.7, 3.0, 3.1, or 3.2. '
                             'When using Python 2.7, please install IPython 5.x LTS Long Term Support version. '
                             'Beginning with IPython 6.0, Python 3.3. and above is required.') 
```

Since the error message will be going to someone that does not have 3.3+ installed, the above error message using the following: 1. Tell them the error. 2. Tell them the resolution (use 5.x LTS). 3. Tell them why.